### PR TITLE
Clarify documentation on password requirements

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

Adds a line to front-end explanation of password requirements to clarify what is expected from the users.

This is a small improvement but we need to add #5761 in order for this to be truly user-friendly.

#### Link(s) to related issue(s)

* There was some confusion about the password requirements resulting in issue #5962. This is meant to address that confusion.

### CODE REVIEW

**Comment by thomasbeaudry** [comment]
Well it's better but it could be clearer.  What does "sufficiently complex" mean?  Can you provide a link to the standards if a user wants more info?  Or if it's short enough can you summarize what is required?

**Comment by johnsaigle** [comment]
There are no hard rules for standards which makes this tough to word.

We're using Zxcvbn to handle a few NIST requirements spelled out here https://github.com/aces/Loris/issues/3314

> Must compare passwords against a blacklist of known passwords when changing
(examples of blacklists given in is passwords from previous breach corpuses,
dictionary words, repetitive or sequential characters, context-specific words
such as the name of the service, username or derivatives thereof.)
>  If password appears in a blacklist, force a different password
> No complexity requirements other than the length

My interpretation of this is basically that we can require that passwords be of at least X characters in length (8, currently) but beyond that we cannot require any specific character set. 

Zxcvbn helps us handle the first point above by checking against those lists and substitutions.

I don't know how to word in a general way that one's password has to be just.... not "bad". The password `9YgR4%(#` is valid and so is `correct horse battery staple` but the only thing they have in common is that they have entropy/complexity.

File: modules/login/templates/form_passwordexpiry.tpl [hunk 1]
```diff
@@ -7,8 +7,8 @@
       <p><b>Password Strength Rules</b></p>
       <ul>
         <li>The password must be at least 8 characters long.</li>
-        <li>The password and the user name must not be the same.</li>
         <li>The password and the email address must not be the same.</li>
+        <li>No special characters are required but your password must be sufficiently complex to be accepted.</li>
```
Comment (PapillonMcGill) [inline modules/login/templates/form_passwordexpiry.tpl]: Can you clarify what are the condition for "sufficiently complex" password?

Someone might think that `9876SA321` is enough

Comment (gdevenyi) [reply modules/login/templates/form_passwordexpiry.tpl]: Its not really possible to provide strict requirements as those are mostly meaningless compared to the entropy calculations provided by Zxcvbn.

When Zxcvbn is fully integrated on the frontend it will provide live feedback on on the password and show when it "passes" the minimum requirements.

Comment (johnsaigle) [reply modules/login/templates/form_passwordexpiry.tpl]: @PapillonMcGill see my above comment. There isn't really a way to explain it better. I agree we really need the front-end Zxcvbn module #5761 which is held up by #5828

Comment (PapillonMcGill) [reply modules/login/templates/form_passwordexpiry.tpl]: @johnsaigle maybe add 
"... sufficiently complex and hard to guest to be accepted."

Comment (johnsaigle) [reply modules/login/templates/form_passwordexpiry.tpl]: What is "hard to guess" though?

Comment (PapillonMcGill) [reply modules/login/templates/form_passwordexpiry.tpl]: I know, not easy. Ok, let go with it, better than current.
